### ISSUE TITLE
Invoking 'share team project' doesn't offer any repository plugins

### ISSUE THREAD

**Issue by majkinetor** [serge-rider/dbeaver#614]
![image](https://cloud.githubusercontent.com/assets/85767/16987456/a4bc7d90-4e8c-11e6-860a-c8053e27e9d4.png)

**Comment by serge-rider** [comment]
Project team sharing isn't yet fully supported. It should include version control systems integration (SVN, Git, etc). And at the moment DBeaver doesn't have any.
Hopefully this feature will be ready at Autumn.

**Comment by majkinetor** [comment]
Why don't you allow that project has it own project file associated with dbeaver .. Then simply copying a directory to a team member (or getting it from a repository) and opening project file would open a project (IDE normally works this way). This would require that all project dependent stuff be in its directory such as drivers used etc. 

So one can use whatever repository wants for this and if you pull the project and have local changes you will have to eventually merge it yourself.

In other words, are the team options really worth implementing when you can already achieve all that by specialized tool that everybody already uses.

**Comment by serge-rider** [comment]
Not sure I got it.

Anyhow, idea is to include existing Eclipse plugins (SVN, Git) in DBeaver.

Currently you can do it by using existing external tools (e.g. command line). DBeaver project is just a regular Eclipse project, you can put it under version control, copy it, etc.

**Comment by majkinetor** [comment]
Currently you have `.project` file. To load a project X you have to import it. Why not simply name a project file `X.dbproject`   so we don't have to import stuff, just double click it and you are ready to work. Its cumbersome like this IMO.
